README.md

```diff
@@ -12,7 +14,6 @@ This template supports debugging using any of the following techniques:
 
 - [Use a browser's developer tools](https://docs.microsoft.com/office/dev/add-ins/testing/debug-add-ins-in-office-online)
 - [Attach a debugger from the task pane](https://docs.microsoft.com/office/dev/add-ins/testing/attach-debugger-from-task-pane)
-- [Use F12 developer tools on Windows 10](https://docs.microsoft.com/office/dev/add-ins/testing/debug-add-ins-using-f12-developer-tools-on-windows-10)
```

Comment:
Same as the other repo, why are we removing this?

Reply:
Specific to IE support.